### PR TITLE
fixed/nav

### PR DESCRIPTION
**Pull request by AdemBenAbdallah**

### CODE REVIEW

File: src/component/Personalize/index.jsx
Comment (TomIA24) [inline src/component/Personalize/index.jsx:1]: in line 1026 (and in general) when we use states, variables the name shoud be in camelCase format

**Review by TomIA24 — CHANGES_REQUESTED** [review]

File: src/component/Nav/index.jsx [hunk 1]
```diff
@@ -61,7 +83,9 @@ const Nav = () => {
   };
 
   const closepopup = () => {
-    navigate("/candidate/profile");
+    console.log('close popup');
```
Comment (elayeboussama) [inline src/component/Nav/index.jsx:86]: don't forget to remove console.log after testing

File: src/component/Nav/index.jsx [hunk 2]
```diff
@@ -61,7 +83,9 @@ const Nav = () => {
...
+    
+    navigate('/candidate/profile');
     setpopupopen(false);
```
Comment (elayeboussama) [inline src/component/Nav/index.jsx:89]: change it to setPopupOpen ( camel case naming for variables)

File: src/component/Nav/index.jsx [hunk 3]
```diff
@@ -444,7 +468,7 @@ const Nav = () => {
               <div className={styles.closbutton}>
                 {" "}
```
Comment (elayeboussama) [inline src/component/Nav/index.jsx:469]: remove this

**Review by elayeboussama — CHANGES_REQUESTED** [review]

File: src/component/Personalize/index.jsx
Comment (Nesrine1333) [reply src/component/Personalize/index.jsx:1]: Done.

File: src/component/Nav/index.jsx (hunk 3 above)
Comment (Nesrine1333) [reply src/component/Nav/index.jsx]: Done

**Pull request merged**